packages/framework/src/Form/CustomerType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CustomerType extends AbstractType
```

Comment:
imo it should be DisplayOnlyCustomerType, to be consistent with other DisplayOnly... types

Reply:
And it should be described in [the docs](https://github.com/shopsys/shopsys/blob/v7.0.0-beta2/docs/wip_glassbox/form-extension.md#what-can-we-use-for-creating-our-own-forms).